src/lib/EasyCustom.jsx

```diff
@@ -44,3 +45,21 @@ export default class EasyCustom extends Component {
     );
   }
 }
+
+EasyCustom.propTypes = {
+  children: PropTypes.element,
+  cssClassPrefix: PropTypes.string,
+  onBlur: PropTypes.func,
+  onFocus: PropTypes.func,
+  setValue: PropTypes.func,
+  value: PropTypes.string,
+}
+
+EasyCustom.defaultProps = {
```

Comment:
As no props are marked as required, there is no reason to have default props declared. Can you remove this section?